charts/devtron/devtron-bom.yaml

```diff
@@ -9,13 +9,14 @@ global:
     runAsNonRoot: true
 
 installer:
-  release: "v0.6.26"
+  release: "v0.6.27"
   image: "quay.io/devtron/inception"
-  tag: "44b30917-185-13275"
+  tag: "a9820f38-185-20840"
+  openshift: false # Set this to true if you are installing on openshift
```

Comment:
Do we need this here?

Reply:
Won't this reset openshift to false on next update from UI for user that enabled it initially?